index.d.ts

```diff
@@ -0,0 +1,17 @@
+import {Node, Parent} from 'unist'
```

Comment:
typings should usually note the minimum supported typescript version.
In this case it depends on unist-util-is which needs at least `3.5`:

```suggestion
// TypeScript Version: 3.5

import {Node, Parent} from 'unist'
```